src/app/common/components/AppBar/AppBar.tsx

```diff
@@ -170,10 +168,8 @@ const AppBarComponent = () => {
                           <Link href={joinSsvLink} target="_blank">
                             <Button disable={false} type={'primary'} text={'Join SSV Network'} />
                           </Link>
-                          {isPrater && (
```

Comment:
if you remove this condition it will show this button also for mainnet, as I remember we shouldn't show it on mainnet network, you need to add new condition to show button for testnets only

Reply:
sync about it with Ariel

Reply:
Ariel:
It should change according to network
E.g. holesky prater mainnet